_docs_use-cases/use-case-consumption-delete-a-repositoryattribute.md

```diff
@@ -0,0 +1,60 @@
+---
+# Start automatic generation
+permalink: use-case-consumption-delete-a-repositoryattribute
+published: true
+title: "Delete a RepositoryAttribute"
+type: use-case
+toc: true
+sidebar:
+  - title: "Integrate Enmeshed"
+    nav: "docs_integrate"
+properties:
+  - id: RA7
+  - component: Runtime
+  - layer: Consumption
+  - facade: AttributesFacade
+  - function: deleteRepositoryAttribute
+  - description:
+  - feature category: Normalized attributes
+  - tech category: Attributes
+  - status: DONE
+  - documentation status:
+  - comments:
+  - actor: Identity
+  - trigger: REST API
+  - precondition:
+  - result:
+  - priority: n/a
+  - complexity: n/a
+  - size: n/a
+  - created_at:
+  - changed_at:
+  - api_route_regex:
+  - published: default
+  - link: use-case-consumption-delete-a-repositoryattribute
+require:
+required_by:
+# End automatic generation
+---
+
+{{properties.description}}
+
+{% include properties_list.html %}
+
+This use-case allows you to delete a RepositoryAttribute, i.e. an own [LocalAttribute]({%link _docs_integrate/data-model-overview.md %}#localattribute) without `shareInfo`.
+
+## Parameters
+
+- The `attributeId` of the RepositoryAttribute you want to delete.
+
+## On Success
+
+- The RepositoryAttribute will be deleted.
+- All predecessors of the RepositoryAttribute will be deleted.
+- If the RepositoryAttribute was succeeded, the `succeeds` property of the successor will be set to undefined.
+- If there are shared copies of the RepositoryAttribute, the `shareInfo.sourceAttribute` of those own shared Attributes and their predecessors will be set to undefined.
```

Comment:
More precise would be: "If there are shared copies of the RepositoryAttribute **or any eventually existing predecessor of it**, the `shareInfo.sourceAttribute` of those own shared Attributes will be set to undefined." as it could be the case that there is no own shared Attribute belonging to the RepositoryAttribute itself, but to one of its eventually existing predecessors.